src/event.ts

```diff
@@ -67,13 +68,13 @@ export interface Event {
   mkt_term: string;
   mkt_content: string;
   mkt_campaign: string;
-  contexts: any;
+  contexts: unknown;
```

Comment:
Switching to unknown is a nice idea, but this is a backwards incompatible change. I feel this change pushes this beyond a PATCH release and is at least a MINOR, I appreciate we are still on MAJOR v0 so I think this is fine getting a MINOR bump to 0.2.0.